pkg/services/accesscontrol/resourcepermissions/service.go

```diff
@@ -58,13 +59,19 @@ func New(cfg *setting.Cfg,
 	ac accesscontrol.AccessControl, service accesscontrol.Service, sqlStore db.DB,
 	teamService team.Service, userService user.Service,
 ) (*Service, error) {
+	// TODO: add log, actionsetstore as a dependency
+	log := log.New("accesscontrol.resourcepermissions")
+	actionSetsStore := NewInMemoryActionSets(log)
```

Comment:
The comment here - I think it would be neater to inject the action sets using wire.